app/controllers/events_controller.rb

```diff
@@ -32,7 +32,13 @@ def update
       render :edit
     end
   end
+  
+  def destroy
+    @event =  Event.find(params[:id])
```

Comment:
you're repeating this line multiple times move it to another function and add a before action. [Read this](https://dannemanne.com/posts/ruby_on_rails_explained_before_action)